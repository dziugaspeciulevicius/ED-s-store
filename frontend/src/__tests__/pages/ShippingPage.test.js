import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupTests";
import ShippingPage from "../../pages/ShippingPage.component";
jest.mock("../../pages/ShippingPage.component");

describe("<ShippingPage />", () => {
  it("Renders <ShippingPage /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<ShippingPage />)).toMatchSnapshot();
  });
});
