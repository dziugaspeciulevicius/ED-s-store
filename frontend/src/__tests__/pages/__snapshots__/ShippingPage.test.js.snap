// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShippingPage /> Renders <ShippingPage /> component correctly 1`] = `ShallowWrapper {}`;
